tests/test_pay.py

```diff
@@ -1425,3 +1425,17 @@ def test_setchannelfees(node_factory, bitcoind):
     feerate_ppm_specific = fees_specific[0]['feerate_ppm']
     assert(feerate_base_specific == 1337)
     assert(feerate_ppm_specific == 137)
+
+    # check if negative fees raise error and DB keeps values
+    raised = None
+    try:
+        l1.rpc.setchannelfee(cid, -1, -1)
+    except RpcError as err:
```

Comment:
You can do this with the `with pytest.raises(RpcError)` context much easier. It'll fail if the code within doesn't raise the expected exception and allows matchers on the exceptions.

Reply:
thx, helpful indeed